hashing/main.cpp

```diff
@@ -157,20 +152,19 @@ static int example(mbedtls_platform_context* ctx)
 }
 
 int main() {
-    mbedtls_platform_context platform_ctx;
     int exit_code = MBEDTLS_EXIT_FAILURE;
 
-    if((exit_code = mbedtls_platform_setup(&platform_ctx)) != 0) {
+    if((exit_code = mbedtls_platform_setup(NULL)) != 0) {
```

Comment:
Same review comment as before - remove the assignment to`exit_code`, and initialise it to `MBEDTLS_EXIT_SUCCESS` instead.